.github/workflows/linear-pr-automation.yml

```
@@ -0,0 +1,220 @@
+            const labels = await issue.labels();
+            const parent = await issue.parent;
+
+            // Build PR description
+            let description = pr.body || '';
+
+            // Add Linear metadata section if not present
+            if (!description.includes('## Linear Issue')) {
+              const linearSection = `
+## Linear Issue
+
+- **Issue**: [${issue.identifier}: ${issue.title}](${issue.url})
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 110: did not find expected alphabetic or numeric character

(syntax-check)